CHANGELOG-1.x.md

```diff
@@ -3,6 +3,7 @@
 
 ## vN.N.N
 * Use `Swal.DismissReason` enum in `receiveEnvelope.js` and `Authorizer.js`
+* picklist-field :: Fixed single select field not hydrating data after a successful save.
```

Comment:
@hunyoboy is this specific to single select fields that are using picklists? seems like it could happen to any single select field, given the right configuration

Reply:
@efjacobson that's true. i'm thinking along the lines of `location of issue found : description`.